src/components/NavigationAccount.vue

```diff
@@ -129,16 +129,32 @@ export default {
 		},
 		deleteAccount() {
 			const id = this.account.id
+			OC.dialogs.confirmDestructive(
+				t('mail', 'Fully delete {accountId}\'s account', { accountId: this.account }),
+				t('mail', 'Account deletion'),
```

Comment:
This should be "Remove account" and also be adjusted from "Delete" to "Remove" in the actions menu because here we only _remove_ the account from Nextcloud, not completely _delete_ everything.
```suggestion
				t('mail', 'Remove account'),
```